event-svc/src/event/feed.rs

```diff
@@ -33,7 +33,9 @@ impl ConclusionFeed for EventService {
         limit: i64,
     ) -> anyhow::Result<Vec<ConclusionEvent>> {
         let raw_events = self
-            .fetch_events_since_highwater_mark(highwater_mark, limit)
+            // TODO: Can we make highwater_marks zero based?
```

Comment:
Is this the correct behavior? If we want to fetch events since a highwatermark, we would get events since highwater_mark + 1

Reply:
Yes without this change you always get back the last row from the previous batch as the first row in the next batch.

Reply:
If you look in the code for how the highwater mark is made you can see it already is adding one to it. This is what I mean by saying they are one based. Hence a TODO for later to investigate if we can simplify things to make everything zero based.